Automattic/woocommerce-payments
# Add TTP fees information to timeline view

Fixes #5376 

#### Changes proposed in this Pull Request
This PR updates the transaction view timeline to display the new additional fees for Tap To Pay. 









#### Testing instructions




In order to test this PR you need to checkout `poc-meta-intent-fee` on server side. 

* Order a product with a Cash on delivery payment option.
* Run the mobile app in the simulator. More info on how you can use the mobile app p1674057969318029/1674054134.309279-slack-C04DN7Q51KK and  P91TBi-6Sn-p2 
* Pay the order with Tap to Pay from the mobile application.
* Check the transaction details screen. The TTP fee should be displayed in the timeline

<img width="1015" alt="Screenshot 2023-02-14 at 17 50 24" src="https://user-images.githubusercontent.com/82826872/218788517-085debfa-2d91-49ef-9454-99b421501e85.png">

-------------------


**Post merge**
